backends/csvbackend.py

```diff
@@ -68,52 +60,53 @@ def _read_config(self, settings):
 
     def save(self, namespace, value, timestamp=None):
         """ Store the data in a .csv given a folder. """
-        store_value = pd.DataFrame(columns=['Value', 'Timestamp'])
-        _put_value(value, store_value, timestamp)
-
-        f_name = self.user + '_' + namespace + '.csv'
-        if self.path != '/':
-            f_name = self.path + f_name
-
-        if f_name in os.listdir(self.path):
-            store_value.to_csv(f_name, sep=',', header=False,
-                               mode='a', index=False)
-        else:
-            store_value.to_csv(f_name, sep=',', header=True,
-                               mode='w', index=False)
-
-    def delete(self, file, start=None, end=None):
-        """ Delete a instances of the csv file"""
-        dataframe, file = self._load_file(file)
-        search = _make_search(start, end, dataframe)
-        data_to_drop = dataframe[search]
-        dataframe.drop(data_to_drop.index, inplace=True)
-        if self.path != '/':
-            file = self.path + file
-
-        dataframe.to_csv(file, sep=',', header=True, mode='w', index=False)
-
-    def get(self, file, start=None, end=None, method=None,
-            fill=None, group=None):
-        """Retrieve data from a csv file"""
-        dataframe, file = self._load_file(file)
-        search = _make_search(start, end, dataframe)
+        rows = []
+
+        fname = f"{self.user}_{namespace}.csv"
+        fname = str(Path(self.path, fname))
```

Comment:
Do you really need to convert it to `str`?

Reply:
Yes.